diff --git a/dist/_chunks/trace.mjs b/dist/_chunks/trace.mjs
index f4bd23779dd4106a48ebeb74cfe7cde72eb8258a..2eee39b1b0e9a9275c6b0800a1bd3b053cef5547 100644
--- a/dist/_chunks/trace.mjs
+++ b/dist/_chunks/trace.mjs
@@ -3,7 +3,8 @@ import { resolveModulePath } from "./libs/exsolve.mjs";
 import { createRequire } from "node:module";
 import * as fsp from "node:fs/promises";
 import { access, readFile, writeFile } from "node:fs/promises";
-import { nodeFileTrace } from "@vercel/nft";
+import __nft_pkg from "@vercel/nft";
+const { nodeFileTrace } = __nft_pkg;
 import semver from "semver";
 var __defProp = Object.defineProperty;
 var __exportAll = (all, no_symbols) => {
